homeassistant/components/mealie/__init__.py

```diff
@@ -52,7 +52,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: MealieConfigEntry) -> bo
     except MealieConnectionError as error:
         raise ConfigEntryNotReady(error) from error
 
-    if not version.valid or version < MIN_REQUIRED_MEALIE_VERSION:
+    if version.valid and version < MIN_REQUIRED_MEALIE_VERSION:
```

Comment:
Should we log if someone is running the nightly that they should not use an ancient one?

Reply:
A warning log could get annoying for those always running nightly's who want to, we may get more issues raised about it than not logging.

Reply:
I think my issue is just the wording.  How about
`f"You are running an unknown version of Mealie ({about.version}), this could cause issues."`

Reply:
`It seems like you are using the nightly version of Mealie. The integration could stop working with older nightlies, so please make sure your version is up-to-date`

Just to give people an idea of "oh, this might be it"

Reply:
Done

Reply:
Any update on the logging?  This will need to be merged for 2024.8 otherwise all HASS IO Mealie Addon's will fail to load the integration as they are Nightly.

Reply:
Updated log message to indicate any nightly may not work.